style.css

```diff
@@ -311,7 +311,7 @@ p.featured {
 	margin: 0 auto;
 	display: block;
 }
-h1.page-title {
```

Comment:
This is probably the biggest material change I'm proposing, that we should only have 1 H1 Element on each page, and it shouldn't be the page title. Even then this is 0 visual impact to the reader, but does net us a minor accessibility boost